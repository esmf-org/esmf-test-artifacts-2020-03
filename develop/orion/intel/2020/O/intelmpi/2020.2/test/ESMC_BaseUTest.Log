build time -- 2021-12-21 02:33:11
20211221 035706.741 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211221 035706.742 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211221 035706.742 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211221 035706.742 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211221 035706.742 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211221 035706.742 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211221 035706.742 INFO             PET0 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_05-1-g3e9d170
20211221 035706.743 INFO             PET0 ESMF library build date/time: "Dec 21 2021" "02:31:45"
20211221 035706.743 INFO             PET0 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_O_develop
20211221 035706.743 INFO             PET0 ESMF_COMM                   : intelmpi
20211221 035706.745 INFO             PET0 ESMF_MOAB                   : enabled
20211221 035706.745 INFO             PET0 ESMF_LAPACK                 : enabled
20211221 035706.745 INFO             PET0 ESMF_NETCDF                 : enabled
20211221 035706.745 INFO             PET0 ESMF_PNETCDF                : disabled
20211221 035706.753 INFO             PET0 ESMF_PIO                    : enabled
20211221 035706.753 INFO             PET0 ESMF_YAMLCPP                : enabled
20211221 035706.754 INFO             PET0 Beginning Test, file ESMC_BaseUTest.C, line 143
20211221 035706.755 INFO             PET0 NUMBER_OF_PROCESSORS 4
20211221 035706.793 INFO             PET0 testSerializeDeserialize(): inquire_offset=869
20211221 035706.793 INFO             PET0 testSerializeDeserialize(): offset=829
20211221 035706.794 INFO             PET0 testSerializeDeserialize(): inquire_offset=936
20211221 035706.794 INFO             PET0 testSerializeDeserialize(): offset=896
20211221 035706.794 INFO             PET0 PASS testSerializeDeserialize, ESMC_BaseUTest.C, line 150
20211221 035706.794 INFO             PET0 Ending Test, file ESMC_BaseUTest.C, line 154
20211221 035706.794 INFO             PET0 Finalizing ESMF
20211221 035706.741 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211221 035706.742 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211221 035706.742 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211221 035706.742 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211221 035706.742 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211221 035706.742 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211221 035706.742 INFO             PET1 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_05-1-g3e9d170
20211221 035706.743 INFO             PET1 ESMF library build date/time: "Dec 21 2021" "02:31:45"
20211221 035706.743 INFO             PET1 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_O_develop
20211221 035706.743 INFO             PET1 ESMF_COMM                   : intelmpi
20211221 035706.745 INFO             PET1 ESMF_MOAB                   : enabled
20211221 035706.745 INFO             PET1 ESMF_LAPACK                 : enabled
20211221 035706.745 INFO             PET1 ESMF_NETCDF                 : enabled
20211221 035706.745 INFO             PET1 ESMF_PNETCDF                : disabled
20211221 035706.753 INFO             PET1 ESMF_PIO                    : enabled
20211221 035706.753 INFO             PET1 ESMF_YAMLCPP                : enabled
20211221 035706.754 INFO             PET1 Beginning Test, file ESMC_BaseUTest.C, line 143
20211221 035706.755 INFO             PET1 NUMBER_OF_PROCESSORS 4
20211221 035706.793 INFO             PET1 testSerializeDeserialize(): inquire_offset=869
20211221 035706.793 INFO             PET1 testSerializeDeserialize(): offset=829
20211221 035706.794 INFO             PET1 testSerializeDeserialize(): inquire_offset=936
20211221 035706.794 INFO             PET1 testSerializeDeserialize(): offset=896
20211221 035706.794 INFO             PET1 PASS testSerializeDeserialize, ESMC_BaseUTest.C, line 150
20211221 035706.794 INFO             PET1 Ending Test, file ESMC_BaseUTest.C, line 154
20211221 035706.794 INFO             PET1 Finalizing ESMF
20211221 035706.741 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211221 035706.742 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211221 035706.742 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211221 035706.742 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211221 035706.742 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211221 035706.742 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211221 035706.742 INFO             PET2 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_05-1-g3e9d170
20211221 035706.743 INFO             PET2 ESMF library build date/time: "Dec 21 2021" "02:31:45"
20211221 035706.743 INFO             PET2 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_O_develop
20211221 035706.743 INFO             PET2 ESMF_COMM                   : intelmpi
20211221 035706.745 INFO             PET2 ESMF_MOAB                   : enabled
20211221 035706.745 INFO             PET2 ESMF_LAPACK                 : enabled
20211221 035706.745 INFO             PET2 ESMF_NETCDF                 : enabled
20211221 035706.745 INFO             PET2 ESMF_PNETCDF                : disabled
20211221 035706.753 INFO             PET2 ESMF_PIO                    : enabled
20211221 035706.753 INFO             PET2 ESMF_YAMLCPP                : enabled
20211221 035706.754 INFO             PET2 Beginning Test, file ESMC_BaseUTest.C, line 143
20211221 035706.755 INFO             PET2 NUMBER_OF_PROCESSORS 4
20211221 035706.793 INFO             PET2 testSerializeDeserialize(): inquire_offset=869
20211221 035706.793 INFO             PET2 testSerializeDeserialize(): offset=829
20211221 035706.794 INFO             PET2 testSerializeDeserialize(): inquire_offset=936
20211221 035706.794 INFO             PET2 testSerializeDeserialize(): offset=896
20211221 035706.795 INFO             PET2 PASS testSerializeDeserialize, ESMC_BaseUTest.C, line 150
20211221 035706.795 INFO             PET2 Ending Test, file ESMC_BaseUTest.C, line 154
20211221 035706.801 INFO             PET2 Finalizing ESMF
20211221 035706.741 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211221 035706.742 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211221 035706.742 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211221 035706.742 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211221 035706.742 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211221 035706.742 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211221 035706.742 INFO             PET3 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_05-1-g3e9d170
20211221 035706.743 INFO             PET3 ESMF library build date/time: "Dec 21 2021" "02:31:45"
20211221 035706.743 INFO             PET3 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_O_develop
20211221 035706.743 INFO             PET3 ESMF_COMM                   : intelmpi
20211221 035706.745 INFO             PET3 ESMF_MOAB                   : enabled
20211221 035706.745 INFO             PET3 ESMF_LAPACK                 : enabled
20211221 035706.745 INFO             PET3 ESMF_NETCDF                 : enabled
20211221 035706.745 INFO             PET3 ESMF_PNETCDF                : disabled
20211221 035706.753 INFO             PET3 ESMF_PIO                    : enabled
20211221 035706.753 INFO             PET3 ESMF_YAMLCPP                : enabled
20211221 035706.754 INFO             PET3 Beginning Test, file ESMC_BaseUTest.C, line 143
20211221 035706.755 INFO             PET3 NUMBER_OF_PROCESSORS 4
20211221 035706.793 INFO             PET3 testSerializeDeserialize(): inquire_offset=869
20211221 035706.793 INFO             PET3 testSerializeDeserialize(): offset=829
20211221 035706.794 INFO             PET3 testSerializeDeserialize(): inquire_offset=936
20211221 035706.794 INFO             PET3 testSerializeDeserialize(): offset=896
20211221 035706.795 INFO             PET3 PASS testSerializeDeserialize, ESMC_BaseUTest.C, line 150
20211221 035706.795 INFO             PET3 Ending Test, file ESMC_BaseUTest.C, line 154
20211221 035706.795 INFO             PET3 Finalizing ESMF
