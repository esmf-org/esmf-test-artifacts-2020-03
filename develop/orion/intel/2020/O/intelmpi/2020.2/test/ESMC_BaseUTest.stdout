build time -- 2021-10-05 02:41:49
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
Beginning Test, file ESMC_BaseUTest.C, line 143
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_BaseUTest.C, line 143
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_BaseUTest.C, line 143
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_BaseUTest.C, line 143
NUMBER_OF_PROCESSORS 4
PASS testSerializeDeserialize, ESMC_BaseUTest.C, line 150
Ending Test, file ESMC_BaseUTest.C, line 154
PASS testSerializeDeserialize, ESMC_BaseUTest.C, line 150
Ending Test, file ESMC_BaseUTest.C, line 154
PASS testSerializeDeserialize, ESMC_BaseUTest.C, line 150
Ending Test, file ESMC_BaseUTest.C, line 154
PASS testSerializeDeserialize, ESMC_BaseUTest.C, line 150
Ending Test, file ESMC_BaseUTest.C, line 154
 PET 1 Test Elapsed Time  115.400000 msec.
 PET 2 Test Elapsed Time  112.809000 msec.
 PET 0 Test Elapsed Time  115.113000 msec.
 PET 3 Test Elapsed Time  113.293000 msec.
